libs/wingsdk/src/cloud/api.ts

```diff
@@ -46,6 +46,12 @@ export interface ApiCorsOptions {
    * @default - false
    */
   readonly allowCredentials?: boolean;
+
+  /**
+   * How long the browser should cache preflight request results.
+   * @default - 300 seconds
+   */
+  readonly maxAge?: Duration;
```

Comment:
Can we update the unit tests to validate configuring this option works / the default works? https://github.com/winglang/wing/blob/b71cae19d036459e5fee1a6084e229bd34bc7a01/libs/wingsdk/test/target-sim/api.test.ts#L601-L663

I think since the value is a `Duration` you'll need to call `.seconds()` on it in the implementation, or something like that.

Reply:
I added the test under the preflight request (`OPTIONS` method).